bin/handlemail

```diff
@@ -161,14 +171,15 @@ sub handle_non_bounce_to_null_address {
     # Send an automatic response
     print_log('info', "Received non-bounce to null address, auto-replying");
     my $template = 'reply-autoresponse';
-    my $fp = FixMyStreet->path_to("templates", "email", "default", $template)->open or exit 75;
+    my $fp = FixMyStreet->path_to("templates", "email", $cobrand, $template)->open or exit 75;
     $template = join('', <$fp>);
     $fp->close;
 
     # We generate this as a bounce.
     my ($rp) = $data{return_path} =~ /^\s*<(.*)>\s*$/;
     my $mail = FixMyStreet::Email::construct_email({
-        From => [ FixMyStreet->config('CONTACT_EMAIL'), 'FixMyStreet' ],
+        From => [ FixMyStreet->config('CONTACT_EMAIL'),
+                  FixMyStreet->config('CONTACT_NAME') ],
```

Comment:
Note that this is potentially a change in behaviour even if the command line options aren't used.

But I guess this is how we want it? The UK site probably has CONTACT_NAME = FixMyStreet anyway?

Reply:
Yes, that looks fine to me!